### PR TITLE
[tw] Add content/zh-tw/abstraction.md

### LINKED ISSUE #2152: [tw] Localize `Abstraction` for Traditional Chinese
+ Localize `Abstraction` in Traditional Chinese
+ https://glossary.cncf.io/abstraction/

### PR DESCRIPTION
**Pull request by gamerslouis**
### Describe your changes
Origin content from https://glossary.cncf.io/abstraction/

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2152 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/abstraction.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: 抽象
+status: Completed
+category: 特性
+tags: ["基本原理", "", ""]
+---
+
+在電腦運算的語境中，抽象是一種表示方式，它將細節隱藏起來，讓[服務](/service/)的使用者（包括電腦程式和人類）能夠更容易理解系統並使其更通用。一個很好的例子是你的筆記型電腦的作業系統（OS）。它抽象化你的電腦運作的所有細節。你不需要了解中央處理器、記憶體以及程式如何運作，你只需操作作業系統，作業系統會處理這些細節。所有細節都被隱藏在作業系統的「幕後」或抽象之後。
```
Comment (pichuang) [inline content/zh-tw/abstraction.md:8]:
```suggestion
在計算機的背景中，抽象是一種表示方式，它將細節隱藏起來，讓[服務](/zh-tw/service/)的使用者（包括電腦程式和人類）能夠更容易理解系統並使其更通用。一個很好的例子是你的筆記型電腦的作業系統（OS）。它抽象化你的計算機運作的所有細節。你不需要了解中央處理器、記憶體以及程式如何運作，你只需操作作業系統，作業系統會處理這些細節。所有細節都被隱藏在作業系統的「幕後」或抽象之後。
```

**Comment by pichuang** [comment]
Hi @gamerslouis First of all, I'd like to thank you for your PR. Based on the principle of translating one glossary at a time, it is recommended to complete the `abstraction` first before processing it to the next glossary. We hope you will complete the revision of the glossary or provide other suggestions in the next review cycle. 😃

**Comment by gamerslouis** [comment]
> Hi @gamerslouis If you are not sure about the above changes, you can go to slack [#glossary-localisation-tranditional-chinese](https://cloud-native.slack.com/archives/C055KERTNCX) to discuss.

Sorry, I missed the notification for this PR, and I don't have any questions!

**Review by pichuang — APPROVED** [review]
lgtm. Please rebase 2 commits into 1 before merging.

https://docs.github.com/en/get-started/using-git/using-git-rebase-on-the-command-line

**Review by hwchiu — APPROVED** [review]

File: content/zh-tw/abstraction.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: 抽象
+status: Completed
+category: 特性
+tags: ["基本原理", "", ""]
+---
+
+在計算機的背景中，抽象是一種表示方式，它將細節隱藏起來，讓[服務](/zh-tw/service/)的使用者（包括電腦程式和人類）能夠更容易理解系統並使其更通用。一個很好的例子是你的筆記型電腦的作業系統（OS）。它抽象化你的計算機運作的所有細節。你不需要了解中央處理器、記憶體以及程式如何運作，你只需操作作業系統，作業系統會處理這些細節。所有細節都被隱藏在作業系統的「幕後」或抽象之後。
```
[the remark this answers is not in the document]
Comment (ydFu) [reply content/zh-tw/abstraction.md:8]: In the context of traditional Chinese usage, the term "CPU" is typically retained without translation.

**Review by ydFu — COMMENTED** [review]
In accordance with the [style guide](https://glossary.cncf.io/style-guide/#three-subheadings) set forth by the cncf/glossary,please use semantic line breaks (one sentence per line)

File: content/zh-tw/abstraction.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: 抽象
+status: Completed
+category: 特性
+tags: ["基本原理", "", ""]
+---
+
+在計算機的背景中，抽象是一種表示方式，它將細節隱藏起來，
+讓[服務](/zh-tw/service/)的使用者（包括電腦程式和人類）能夠更容易理解系統並使其更通用。
+一個很好的例子是你的筆記型電腦的作業系統（OS）。
```
Comment (ydFu) [inline content/zh-tw/abstraction.md:10]:
```suggestion
一個很好的例子是您的筆記型電腦的作業系統（OS）。
```

File: content/zh-tw/abstraction.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+它抽象化你的計算機運作的所有細節。
```
Comment (ydFu) [inline content/zh-tw/abstraction.md:11]:
```suggestion
它抽象化您的計算機運作的所有細節。
```

File: content/zh-tw/abstraction.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+你不需要了解中央處理器、記憶體以及程式如何運作，你只需操作作業系統，作業系統會處理這些細節。
```
Comment (ydFu) [inline content/zh-tw/abstraction.md:12]:
```suggestion
您不需要了解 CPU、記憶體以及程式如何運作，
您只需操作作業系統，作業系統會處理這些細節。
```

File: content/zh-tw/abstraction.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+所有細節都被隱藏在作業系統的「幕後」或抽象之後。
+
+系統通常具有多個抽象層。這顯著簡化了開發過程。
```
Comment (ydFu) [inline content/zh-tw/abstraction.md:15]:
```suggestion
系統通常具有多個抽象層。
這顯著簡化了開發過程。
```

File: content/zh-tw/abstraction.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+在程式設計時，開發人員建立與特定抽象層兼容的元件，而不需要擔心底層的具體細節及差異。
```
Comment (ydFu) [inline content/zh-tw/abstraction.md:16]:
```suggestion
在程式設計時，開發人員建立與特定抽象層兼容的元件，
而不需要擔心底層的具體細節及差異。
```

**Review by ydFu — COMMENTED** [review]
It is recommended to change the line to follow the English upstream and it is easy to maintain when the subsequent English upstream changes.

**Review by ydFu — APPROVED** [review]
LGTM,thanks @gamerslouis

**Comment by ydFu** [comment]
> lgtm. Please rebase 2 commits into 1 before merging.
> 
> https://docs.github.com/en/get-started/using-git/using-git-rebase-on-the-command-line

@gamerslouis 
Use rebase to squash/fixup dummy/unnecessary commits into only one commit.

**Pull request merged**